help/destinations/catalog/crm/zendesk.md

```diff
@@ -0,0 +1,166 @@
+
+### Authenticate to destination {#authenticate}
+
+Fill in the required fields below. Refer to the [Gather [!DNL Zendesk] credentials](#gather-credentials) section for any guidance.
+* **[!UICONTROL Bearer Token]**: The Access Token you have generated in your [!DNL Zendesk] account.
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
+![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/zendesk/authenticate-destination.png)
+
+If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark. You can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Platform UI screenshot showing the destination details.](../../assets/catalog/crm/zendesk/destination-details.png)
+
+* **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Zendesk] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination.
+
+>[!IMPORTANT]
+>
+>Both the attribute mappings listed in the table which follows are mandatory and required for [!DNL Zendesk] to work.
```

Comment:
This note is confusing because there is no table following it. Either move that part of the note next to the table, or rephrase as "Both attribute mappings listed in the table further below are mandatory and required for [!DNL Zendesk] to work".

Reply:
Changed the position of the note
[Comment](https://github.com/AdobeDocs/experience-platform.en/pull/153#discussion_r1043275221)